docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -179,10 +187,11 @@ kubectl config set-credentials kubelet-bootstrap --token=${BOOTSTRAP_TOKEN} --ku
 
 When starting the kubelet, if the file specified by `--kubeconfig` does not exist, the bootstrap kubeconfig is used to request a client certificate from the API server. On approval of the certificate request and receipt back by the kubelet, a kubeconfig file referencing the generated key and obtained certificate is written to the path specified by `--kubeconfig`. The certificate and key file will be placed in the directory specified by `--cert-dir`.
 
-The flag to enable this bootstrapping when starting the kubelet is:
+The following flags are required to enable this bootstrapping when starting the kubelet is:
```

Comment:
"Note:" is what the kubeadm docs do https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/setup/independent/install-kubeadm.md#installing-kubelet-and-kubeadm

"Note: The following flags..." sgtm